doc/developer-notes.md

```diff
@@ -1116,31 +1116,6 @@ TRY_LOCK(cs_vNodes, lockNodes);
 }
 ```
 
-Scripts
---------------------------
-
-Write scripts in Python rather than bash, when possible.
```

Comment:
fa220ec193436fd1e62e29a8f3525bf26fa3c209: this is still useful, even if often ignored :-)

Reply:
Not to bash rust, but maybe suggest either...

Reply:
> Write scripts in Python or Rust rather than bash, when possible.

*technically* only Python is a "scripting language" for writing "scripts".  I'm not sure you could say programs written in Rust are scripts,  although I'm sure people will know what is intended by this sentence.